Add better examples around safe private data upload (#244)
encord-team/encord-client-python: docs/source/tutorials/datasets.rst

        2. A way to enable cloud integrations from the SDK?
        3. Sample outputs of the given script

.. note::

>     We strongly encourage you to follow and adapt the below script for a safe way to upload data to your dataset.

Reviewer: ```suggestion
    We strongly encourage you to follow and adapt the below script for an idiomatic way to upload data to your dataset.
```
Replace safe with idiomatic. Let's not ask people why it might not be 'safe' :D
